features/referent-organizational-chart.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Make sure we can see and interact with organizational chart
+  In order to have an organizational chart
+  As a root referent
+  I need to be able to see and edit an organizational chart
+
+  Scenario: As a root referent I can see and click to access to organizational chart page
+    Given the following fixtures are loaded:
+      | LoadAdherentData |
+      | LoadOrganizationalChartItemData |
+    And I am logged as "referent@en-marche-dev.fr"
+    And I am on "/espace-referent/utilisateurs"
+    Then I should see "Equipe départementale"
+    When I follow "Equipe départementale"
+    Then I should see "Référent départemental"
+    And I should see "Responsable logistique -"
```

Comment:
IMO the ` -` character is useless for this test

Reply:
mmmmmmmmmmmmm no, I want to be sure there are nobody link to this node.